src/modules/fw_att_control/fw_att_control_params.c

```diff
@@ -528,17 +528,12 @@ PARAM_DEFINE_FLOAT(FW_FLAPS_SCL, 1.0f);
 PARAM_DEFINE_FLOAT(FW_FLAPERON_SCL, 0.0f);
 
 /**
- * Airspeed mode
+ * Disable airspeed sensor
  *
- * The param value sets the method used to publish the control state airspeed.
  * For small wings or VTOL without airspeed sensor this parameter can be used to
  * enable flying without an airspeed reading
  *
- * @min 0
- * @max 2
- * @value 0 use measured airspeed
- * @value 1 use vehicle ground velocity as airspeed
- * @value 2 declare airspeed invalid
+ * @boolean
  * @group FW Attitude Control
```

Comment:
In vtol it's copied then constrained [0, 1], so existing setups without airspeed should be safe and I can't imagine anyone is actually using the groundspeed option.